compiler_opt/rl/env.py

```diff
@@ -311,9 +313,12 @@ def _get_clang_generator(
     module = yield
     with clang_session(
         clang_path, module, task_type, interactive=True) as iclang:
-      with clang_session(
-          clang_path, module, task_type, interactive=False) as clang:
-        yield iclang, clang
+      if interactive_only:
+        yield iclang, iclang
```

Comment:
Could we return iclang here, i.e. not a tuple, and detect that on the receiving end? or (iclang, None)?

Reply:
The relative reward is computed, that is, the compute_relative_rewards function of env.py always expects two scores. I found this to be the cleanest change. Returning only iclang and None will end up being identical but would require multiple changes and I didn't want to break anything.

Reply:
OK - can you update the documentation string explaining this, too? (i.e. it's not quite "`interactive_only: If set to true only iclang is yielded`" - looking at the docstring, the return isn't quite well specified actually, what the generator generates is what's interesting - could you basically make the doc "right"?)